scripts/build.mjs

```diff
@@ -0,0 +1,69 @@
+import chalk from 'chalk'
+import { build } from 'esbuild'
+import { existsSync, readFileSync } from 'fs'
+import { join } from 'path'
+import walk from 'walkdir'
+
+const ignorePackages = ['teleport-repl-component', 'teleport-test']
+const emitter = walk('packages', {
+  max_depth: 1,
+})
+
+emitter.on('directory', async (path) => {
+  const split = path.split('/')
+  const packageName = split[split.length - 1]
+  if (ignorePackages.includes(packageName)) {
+    return
+  }
+
+  await buildPackage(path, packageName)
+})
+emitter.on('error', (err) => console.log(err))
+
+export const buildPackage = async (path, packageName) => {
+  const entry = `${path}/src/index.ts`
+  const isEntryExists = existsSync(entry)
+  let packageJSON
+  try {
+    packageJSON = readFileSync(join(process.cwd(), `packages/${packageName}/package.json`), 'utf-8')
+  } catch (e) {
+    return
+  }
+
+  if (!isEntryExists || !packageJSON) {
+    throw new Error(`Entry file missing from ${packageName}`)
+  }
+
+  const external = [...Object.keys(JSON.parse(packageJSON)?.dependencies || {})]
+  external.push('path')
+  external.push('fs')
```

Comment:
So, i am specifying `path` and `fs` also as external imports. Even though our main target is browsers but we can run in node too. But if we `polyfill` because of that. It's more of duplication in node environments. 
Anyways we are not polyfilling them before too.

Polyfill happens in which ever environment it is being used. like in repl we do explicitly
https://github.com/teleporthq/teleport-repl/blob/development/next.config.js#L5